schemas/context/segmentdefinition.schema.json

```diff
@@ -0,0 +1,74 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/segmentdefinition",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Segment Definition",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "Segment Definition class to describe a segment from any system",
+  "definitions": {
+    "segmentdefintion": {
+      "properties": {
```

Comment:
This seems to be lacking some important properties like:
-Segment definition expression
-Info about the ims org/sandbox id for which it is defined
-Version
-Things related to how the segment is evaluated like streaming/batch etc
-ttl
-Merge policy

Reply:
These fields will be part of solution sepcific mixin.
That mixin with base class segmentdefinition, will help in creating schema specific to that solution.

Reply:
I think at least following fields should be part of  
- Segment definition expression
- Version
- Info about the IMS org/sandbox id for which it is defined

Reply:
Hey Neeraj, Thanks!

- Segment definition expression is added as mixin and will be added with solution specific extension or schema creation as this is optional for external segments which is owned by other systems.
- Good idea about version, will add it.
- IMS Org Id and Sandbox Id are system level details and not part of schema by default. If they are still required, can be added in solution/system level extensions.